scripts/initscripts/snapd.service

```diff
@@ -0,0 +1,18 @@
+[Unit]
+Description=Starts and stops a single snapd instance on this system
+Documentation=https://github.com/intelsdi-x/snap
+Wants=network-online.target
+After=network-online.target
```

Comment:
network-online.target seems more appropriate. If users have snapd configured to listen on a specific IP, we cant start snapd until that IP is assigned to an interface. And in the case of DHCP, that wont be until the network is up.